man/systemd.network.xml

```diff
@@ -796,6 +796,48 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[RoutingPolicyRule] Section Options</title>
+
+      <para>An <literal>[RoutingPolicyRule]</literal> section accepts the
+      following keys. Specify several <literal>[RoutingPolicyRule]</literal>
+      sections to configure several rules.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>TypeOfService=</varname></term>
+          <listitem>
+            <para>Specifies the type of service to match.</para>
```

Comment:
This needs to say what kind of value is expected here. i.e. format, meanings, range, ...